Azure Blob Storage Collection Update release note (#4094)
SumoLogic/sumologic-documentation: blog-service/2024-02-09-apps.md

---
title: Azure Blob Storage Collection Update (Apps)
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - azure-blob-storage
  - connections
  - webhook connections
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

We're happy to announce an update of the Azure Blob Storage collection framework. Here are some of the key features that this update offers:

- Aligns all the components to the latest version of Azure Functions runtime and libraries.
- Replaces the soon-to-be deprecated classic application insights resource with new workspace-based application insights.
- Updated function application logging with log levels.
- Provides necessary security updates.

> For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). For more information on logs collection from Azure Blob Storage, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage).

Reviewer: ```suggestion
For upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). For information on collecting logs from Azure Blob Storage, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage).
```

> For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).

Reviewer: thanks for confirming!

> For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).

Reviewer: ```suggestion
For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). For more information on logs collection from Azure Blob Storage, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage).
```

> For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).

Reviewer 2: ```suggestion
For details on upgrade instructions, refer to the [changelog](https://github.com/SumoLogic/sumologic-azure-function/releases/tag/v4.0.0). Also, for more information on metrics collection from Azure Monitor, see our [documentation](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/).
```
